test/e2e/config/packet-ci-actions.yaml

```diff
@@ -114,7 +114,7 @@ providers:
       # Add cluster templates
       - sourcePath: "../data/v1alpha3/cluster-template-packet-ccm.yaml"
       - sourcePath: "../data/v1alpha3/cluster-template-cpem.yaml"
-  - name: v0.5.0
+  - name: v0.6.0
     value: "${MANIFEST_PATH:=..}/infrastructure-components.yaml"
     type: "url"
     contract: v1beta1
```

Comment:
If updating `clusterctl-settings.json` for the version number, the entry below for `v1.1.99` would also need to be changed below.